### DISCUSSION TITLE
Buderus GB192i.2 wrong exhaust temperature with Gateway S3 (Standard WiFi Ausgabe V3) Version 3.6.4

### DISCUSSION THREAD

**Discussion by Sancho224** [emsesp/EMS-ESP32#1467] [Q&A]
I have an unrealistic high exhaust temperature "1.638,4°C" all other parameters looks fine for the moment.. I started yesterday with the Gateway S3 (Standard WiFi Ausgabe V3).

MQTT Data transfer 0 failures and WIFI Signal is green all the time.

Do I have to adjust something?? Can anybody help me with that??

**Comment by KLiebrecht1** [comment]
Perhaps it's an high-resolution value? Then divide it by 100.
1.638,4 °C / 100 = 16,384 °C

**Reply by Sancho224** [reply]
it´s all the time the same value now.. since I used this device.. only at the start it begun with 30,4°C for a really short time...

**Reply by MichaelDvP** [reply]
16384 is 0x4000, seems to be a value for no sensor. Normally Bosch uses 0x8000 for open/broken sensor, maybe 0x4000 stands for no input for sensor?

**Reply by Sancho224** [reply]
I checked the manual.. I found nothing about a sensor for exhaust temperature... hmmm.. perhaps there is no sensor for it inside?? I´m not sure anymore...

**Reply by MichaelDvP** [reply]
We had some changes with exhaust temp last time and i see there is a possible issue. Could you check if you get other values with https://github.com/MichaelDvP/EMS-ESP32/releases/tag/test?
A log/read of the telegrams 0x19, 0xE4 and 0xE5 could also help. (in terminal `read 8 19`, `read 8, e4`, `read 8 e5`)

**Reply by Sancho224** [reply]
its my first day of usage.. can I go back to version 3.6.4 after the test version?? will I loose my configuration.. when I make this update??

**Reply by Sancho224** [reply]
I did the firmware update.. now the exhaust temperature is fine... should I stay on this firmware now and wait for the next official firmware realease?? or can I expect some failure with this version??

**Reply by MichaelDvP** [reply]
You can change between all versions, up- or downgrade. Normally you don't loose data.  You can also safe your config (system->up/download) and restore. When changing between major versions there could be changes in the config files, so better check config after update (e.g. 3.4.x -> 3.5.x the names of dallas-temperature-sensors will get lost).
My test build is mainly the same as official dev, only SM100/MM100 modules for hot water are moved from solar/mixer to a new water device.
I'll make a PR to fix the exhausttemp in official dev.

**Reply by Sancho224** [reply]
Thank you for that explanation.. and the working test firmware 👍
